### PR TITLE
streams + attachment size check + attachment url in body

### PR DESCRIPTION
**Pull request by if0s**

### CODE REVIEW

File: component.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "AWS S3",
-  "version": "1.4.3",
+  "version": "2.0.0-dev.2",
```
Comment (shulkaolka) [inline component.json]: please remove .dev

File: package.json [hunk 2]
```diff
@@ -27,9 +27,9 @@
     "elasticio-component"
   ],
   "dependencies": {
-    "@elastic.io/component-commons-library": "2.0.2",
+    "@elastic.io/component-commons-library": "2.1.0-dev.6",
```
Comment (shulkaolka) [inline package.json]: please remove .dev

**Review by shulkaolka — CHANGES_REQUESTED** [review]
Is it necessary to up the major version? (2.0.0)

File: component.json (hunk 1 above)
Comment (if0s) [reply component.json]: after QA, i will keep it in mind

File: package.json (hunk 2 above)
Comment (if0s) [reply package.json]: waiting library release

File: component.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "AWS S3",
-  "version": "1.4.3",
+  "version": "1.5.0-dev.4",
```
Comment (shulkaolka) [inline component.json]: -dev should be deleted

**Review by olegosh — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (if0s) [reply package.json]: done

File: component.json (hunk 3 above)
Comment (if0s) [reply component.json]: after QA

Comment (if0s) [reply component.json]: done

**Pull request merged**